mariadb_kernel/maria_magics/load.py

```diff
@@ -0,0 +1,64 @@
+"""This class implements the %load magic command"""
+
+help_text = """
+The %load magic command has the following syntax:
+    > %load [csv file path] [be updated table name]
+The %load magic command can load CSV file for updating specific table data
+"""
+
+from mariadb_kernel.maria_magics.line_magic import LineMagic
+import pandas
+
+# The class inherits LineMagic because it is a line magic command
+# Cell magic commands have to inherit the CellMagic class
+class Load(LineMagic):
+    def __init__(self, args):
+        temp_args = args.split(" ")
+        self.csv_file_path = ""
+        self.table_name = ""
+        if len(temp_args) == 2:
+            self.csv_file_path = args.split(" ")[0]
+            self.table_name = args.split(" ")[1]
+
+    def name(self):
+        return "%load"
+
+    def help(self):
+        return help_text
+
+    def execute(self, kernel, data):
+        # for error handling
+        if self.csv_file_path == "" or self.table_name == "":
+            err = "argument num must to be 2"
+            kernel._send_message("stderr", err)
+            return
+        try:
+            open(self.csv_file_path)
+        except FileNotFoundError:
+            err = "CSV file not found"
+            kernel._send_message("stderr", err)
+            return
+        # kernel.log.info(json.dumps(self.args))
+        data_frame = pandas.read_csv(self.csv_file_path)
+        cols_info = "(" + ",".join(str(v) for v in data_frame.columns) + ")"
+        rows = [
+            "(" + " ,".join(self.generate_value_str(v) for v in values) + ")"
+            for index, values in data_frame.iterrows()
+        ]
+        rows_info = ",".join(rows)
+        insert_sql = f"INSERT INTO {self.table_name} {cols_info} VALUES {rows_info};"
+        kernel.log.info(insert_sql)
+        insert_result: str = kernel.mariadb_client.run_statement(insert_sql)
```

Comment:
Whilst I'm pretty sure your approach works well, I would like to see a solution based on `LOAD DATA LOCAL INFILE` (https://mariadb.com/kb/en/load-data-infile/). Using `LOAD DATA` will eliminate most of the error handling logic you added here because the server does all the error checking itself and you will only need to check `mariadb_client.iserror()` and if `True` send `mariadb_client.error_message()` string to the frontend.

Please read the KB page for LOAD DATA to understand the advantages of using this query instead of crafting our own.
I also advise we use the LOCAL (so that paths are relative to the kernel location, not the MariaDB server) and the IGNORE keywords. Sometime in the future we can add some args to the magic command to configure these options if users request them.

Reply:
ok, I change the SQL command for updating the table. And use mariadb_client.iserror() to check has an error or not. If it has an error, show it on frontend